assets/javascripts/modules/mma/updatePayment.jsx

```diff
@@ -54,11 +54,12 @@ const WAITING = 'waiting'
 const SUCCESS = 'success'
 const FAILURE = 'failure'
 const FORM = 'form'
+const UNAVAILABLE = 'unavailable'
 
 const CardUpdate = ({ card, handler }) => (<div><button className="button button--primary button--large" onClick={handler}>•••• •••• •••• {card && card.last4 || '••••'} Update Card</button></div>)
 const Success = () => (<p>Thank you, we have successfully updated your payment details.</p>)
 const Failure = ({ phone }) => <p>Unfortunately, we are unable to update your payment details at this time, please contact the call centre. {phone}</p>
-
+const Unavailable = ({ phone }) => (<p>Unfortunately, it is not possible to update your payment details online. To update your payment details, please contact the call centre. {phone}</p>)
```

Comment:
I like the idea of splitting this into a separate message 👍 

But if I understand it correctly, we are still trying to use this one message to cover a lot of cases:

1. We timed out waiting for members-data-api to retrieve the details (problem temporary)
2. members-data-api returned an error when attempting to retrieve the details (problem may be temporary)
3. members-data-api returned a 200, but the response body is empty ([because they don't have an identity account?](https://github.com/guardian/members-data-api/blob/master/membership-attribute-service/app/controllers/AccountController.scala#L197)) (problem is permanent)

Do you think we should refine the error handling further so that we aren't driving unnecessary contacts to the call centre?